psalm.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<psalm
+    errorLevel="8"
```

Comment:
why starting so low? :smile:

Reply:
Didn't look into it too much. There are still a lot of warnings to get rid of. Some by design - that need thorough changes; some are just busy work. But the codebase passes currently

Reply:
done in #109